src/main/java/randoop/output/FailingTestFilter.java

```diff
@@ -67,6 +71,17 @@
   public FailingTestFilter(TestEnvironment testEnvironment, JavaFileWriter javaFileWriter) {
     this.testEnvironment = testEnvironment;
     this.javaFileWriter = javaFileWriter;
+    this.flakyTests = new HashSet<>();
+  }
+
+  /**
+   * Retrieves a shallow copy of the flaky test collection.
+   *
+   * @return a shallow copy of the flaky test collection
+   */
+  @SuppressWarnings("unchecked")
+  public HashSet<String> getFlakyTests() {
+    return (HashSet<String>) flakyTests.clone();
```

Comment:
Use `return new HashSet<>(flakyTests);` because then you don't need to suppress any warnings.  (It's also slightly shorter, for what that's worth.)